### ISSUE TITLE
Error installing on BW6.3.5 due to incorrect

### ISSUE THREAD

**Issue by mcrodrig** [TIBCOSoftware/bw6-plugin-maven#72]
Maven Plugin version: 1.2.1

Maven version: 3.3.9.2

Product : Businessworks 6
Product version: 6.3.5

Component: Installation process

**Steps to reproduce the issue:**
1. Execute the install.bat

**Additional environment details if any:**

**Describe the results you received:**
Error during the installation as the unzip was done in a wrong directory. The unzip was done to 6.3;palettes

**Describe the results you expected:**
Installation process is done correctly

**Additional information you deem important (e.g. issue happens only occasionally):**
I removed the following lines from the build.xml

 			  <elseif>
			  <available file="${tibco.home}/bw" type="dir" />
				<then>
				  <dirset id="bw.version" dir="${tibco.home}/bw" includes="*"/>
				  <property name="prop.bw.version" refid="bw.version"/>
				  <echo message="BW Version ${prop.bw.version}" />
				  <property name="bw.home" value="bw/${prop.bw.version}" />
				</then>
			  </elseif>

**Comment by RajibBiswasTibco** [comment]
@mcrodrig For product version 6.3.5, it should be installed in 6.3 directory. What is the issue here?

**Comment by mcrodrig** [comment]
as I have installed previously a palette, in the <TIBCO_HOME>\bw directory I have directories 6.3 and palettes.
The code I removed set the prop.bw.version to 6.3;palettes. This makes the installer not to work when doing the unzip in the directory <TIBCO_HOME>\bw\6.3\maven\plugins\bw6-maven-plugin

To try, create a directory in TIBCO_HOME\bw and execute in installer if you don't want to install a plugin.

**Comment by alexandrev** [comment]
Hi @RajibBiswasTibco, I could reproduce the issue @mcrodrig opened. 

![image](https://cloud.githubusercontent.com/assets/947041/24478689/0cbfedd0-14dc-11e7-957e-0ebd96913474.png)


And the solution should be to change the includes parameter value of dirset from * to 6.* . 

This is a file from bw 6 so it makes sense to restrict the result to that pattern. But then, you need to fix another issue, because, what happen if you have two major versions of bw installed on the same TIBCO_HOME (think about an upgrade from 6.3.5 to 6.4.0 when it releases) I'm going to have 6.3 and 6.4 folders at that path, right?

So we should only get the higher value of the returned values. This snippet fixes (I'd opened a PR but it seems the build.xml is not managed by this repository. I don't get why...)

			```
	  <dirset id="bw.version" dir="${tibco.home}/bw" includes="6.*"/>
				  <property name="prop.bw.version" refid="bw.version"/>
                  <script language="javascript">
                    arr = project.getProperty('prop.bw.version').split(';');
                    if(arr.length > 0){
                        project.setProperty('prop.bw.version',arr[arr.length-1]);
                    }
                    </script>
				  <echo message="BW Version ${prop.bw.version}" />
				  <property name="bw.home" value="bw/${prop.bw.version}" />
```

**Comment by kckane** [comment]
I ran into this as well. A better solution might be to prompt for the version number if multiple are found.

**Comment by kkane-tibco** [comment]
I ran into this with 6.3.4 as well. Perhaps a better solution would be to prompt for the version number if multiple are found, thus giving developers better control over where the plugin gets installed.

**Comment by RajibBiswasTibco** [comment]
Fixed in Release Version 1.2.2
